src/background.ts

```diff
@@ -1,6 +1,55 @@
+import type { User } from "./types";
+import { createFetcher } from "./utils";
+
+// @ts-ignore
```

Comment:
Can we add https://github.com/kelseasy/web-ext-types or define our own global.d.ts file that does something like `declare namespace browser = chrome` (pseudo code)